src/v2/Apps/Partner/Components/Overview/SubscriberBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Box, Text } from "@artsy/palette"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SubscriberBanner_partner } from "v2/__generated__/SubscriberBanner_partner.graphql"
+
+export interface SubscriberBannerProps {
+  partner: SubscriberBanner_partner
+}
+
+export const SubscriberBanner: React.FC<SubscriberBannerProps> = ({
```

Comment:
Should this be a [Message](https://palette-storybook.artsy.net/?path=/story/components-message--default) component?

Reply:
Do you mean use `Message` component instead of `SubscriberBanner` or  wrapping the layout inside into `Message` would be enough?

I mean something like that

![Screen Shot 2021-05-14 at 5 38 30 PM](https://user-images.githubusercontent.com/79980131/118286521-354cdf80-b4db-11eb-86be-2aac0da22dc7.png)

Reply:
Yeah; in this case you'd pass it the `title` prop for the title; and children for the description.

Reply:
@dzucconi Fixed. Butin this case I will be unable to set the font size for title. It is 24px in figma.

Reply:
Should just use the design system component. The Figma is likely mistaken or incorrect.

Reply:
#neat-picky - but I would name this something like `FullProfileBanner` or something to indicate that this is full page vs limited page. I think we should not tie this to subscription. Some partners like institutions might get it too. And we might change the rules who gets what view to not depend on subscription. This subscription logic should be incapsulated in Gravity/MP. Ui should not encapsulate this rule I believe.

Reply:
Also  - is not this `LimitedPageBanner` - that is what people that are not subscribers get?

Reply:
@oxaudo yes, this component displays on limited overview page so it might be a good idea to name it like `LimitedPageBanner`. On the other hand If I named it LimitedPageBanner then it would mean that we also should have FullPageBanner/FullProfileBanner but we don't :)

 I named it `SubscriberBanner` to emphasize that the banner has some info related to subscribe status.